src/main/java/uk/gov/hmcts/reform/sscs/service/admin/RestoreCasesService2.java

```diff
@@ -147,7 +147,7 @@ public void updateCase(SscsCaseDetails sscsCaseDetails, IdamTokens idamTokens) {
 
         caseData.setProcessingVenue(airLookupService.lookupAirVenueNameByPostCode(postCode, caseData.getAppeal().getBenefitType()));
 
-        String benefitCode = generateBenefitCode(caseData.getAppeal().getBenefitType().getCode());
+        String benefitCode = generateBenefitCode(caseData.getAppeal().getBenefitType().getCode(), "");
```

Comment:
not a great api is it? #whatever!

Reply:
This wasn't my change - there was a breaking commit to common so just added this until another PR was merged. Looks like it's in now so this should disappear from my PR

Reply:
great :)